publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dynamodb/dao/DynamodbStreamRecordImageDao.java

```diff
@@ -122,7 +122,7 @@ public Builder withDynamodbStreamRecordImage(JsonNode rootNode) {
             publicationInstanceType = textFromNode(
                 rootNode,
                 jsonPointers.getEntityDescriptionReferenceTypeJsonPointer());
-            publicationReleaseDate = rootNode.at(jsonPointers.getEntityDescriptionMapJsonPointer());
+            publicationReleaseDate = rootNode.at(jsonPointers.getEntityDescriptionDateJsonPointer());
```

Comment:
PublicationReleaseDatePointer was pointing to EntityDescription and could not parse the date.